tests/integration/individual_response/test_individual_response.py

```diff
@@ -0,0 +1,375 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestIndividualResponse(
+    IntegrationTestCase
+):  # pylint: disable=too-many-public-methods
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey("test_individual_response")
+
+    @property
+    def individual_section_link(self):
+        return self.getHtmlSoup().findAll("a", {"class": "summary__button"})[1]["href"]
+
+    @property
+    def individual_response_link(self):
+        return [
+            td.find("a")
+            for td in self.getHtmlSoup().findAll("div", {"class": "u-mt-s"})
```

Comment:
We shouldn't use classes to locate elements in tests. These should use an `id` or a `data-qa` attribute.